lib/phoenix/controller/logger.ex

```diff
@@ -4,6 +4,10 @@ defmodule Phoenix.Controller.Logger do
   @behaviour Plug
   import Phoenix.Controller
 
+  alias Phoenix.Endpoint.Instrument
+
+  @filter_parameters Application.get_env(:phoenix, :filter_parameters)
```

Comment:
1. No, it is a quick ETS lookup

2. I think we are fine because we call it from two places only. If more start popping up, then we should review it.